.circleci/conditional_config.yml

```diff
@@ -47,7 +47,7 @@ images:
     image: registry.opensuse.org/obs/server/unstable/containers/containers/openbuildservice/frontend-minitest:latest
     <<: *common_frontend_config
     environment:
-      EAGER_LOAD: 1
+      CI: 1
```

Comment:
Why rename this? What's the significance of this being CI or not?

Reply:
Now we attach to the Rails conventions. See: https://github.com/rails/rails/pull/43508